src/style.css

```diff
@@ -1 +1,123 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 8px;
+  font-family: Avenir, sans-serif;
```

Comment:
Good job on defining the font family. However, remember to use fallback fonts - alternative font-family in case the main one doesn't work. This is to ensure that some sort of font is displayed to the user in case the first one doesn't load for some reason. For example, you could modify this line to: `font-family: Avenir, Arial, sans-serif;`